source/options.ts

```diff
@@ -69,6 +69,7 @@ export type QueueAddOptions = {
 	@default 0
 	*/
 	readonly priority?: number;
+	id?: string;
```

Comment:
```suggestion
	readonly id?: string;
```

Reply:
This value is getting updated as per:
https://github.com/RaishavHanspal/p-queue/blob/686f0d8e21efc1cae0b4c855d6a1eab58258d436/source/index.ts#L245